components/Cards/src/MDCCardCollectionCell.h

```diff
@@ -293,3 +307,23 @@ typedef NS_ENUM(NSInteger, MDCCardCellVerticalImageAlignment) {
 @property(nonatomic, readonly) MDCCardCellState state;
 
 @end
+
+/**
+ This protocol is used to integrate the Ripple Beta component into Cards. Please do not conform
+ to this protocol.
+ */
+__deprecated_msg("This protocol is temporarily used to incorporate the Ripple Beta component into Cards. This is deprecated, please do not conform to this protocol.") @protocol MDCCardCollectionCellRippleDelegate <NSObject>
+
+- (void)cardCellRippleDelegateSetSelected:(BOOL)selected;
+- (void)cardCellRippleDelegateSetHighlighted:(BOOL)highlighted;
+- (void)cardCellRippleDelegateSetSelectable:(BOOL)selectable;
+- (void)cardCellRippleDelegateSetDragged:(BOOL)dragged;
+- (UIImage *)cardCellRippleDelegateUpdateImage:(UIImage *)image;
+- (UIColor *)cardCellRippleDelegateUpdateImageTintColor:(UIColor *)imageTintColor;
+- (MDCCardCellState)cardCellRippleDelegateState;
+- (void)cardCellRippleDelegateTouchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event;
+- (void)cardCellRippleDelegateTouchesEnded:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event;
+- (void)cardCellRippleDelegateTouchesCancelled:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event;
+- (void)cardCellRippleDelegateTouchesMoved:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event;
+
+@end
```

Comment:
Similarly here, consider creating a separate beta bridging header that is not included in the component’s umbrella header.

Reply:
Done.